cornell-dti/course-plan
# Modal Refactor Cleanup

### Summary 

This pull request cleans up some issues from the modal refactor (#491) found in [this Notion item](https://www.notion.so/courseplan/82bf355c4b7b49f99691eb9f3afb203c?v=5da9d646f3ef424bbceb1ca0e8e00cf8&p=b14db3e236ba41f4b733aeeade94239d) in the backlog

- [x] Remove all warnings found from classes put on modals that have no affect
- [x] Deleted unused refs, unused modal styling outside of TeleportModal
- [x] Remove some of the z-index toggles remaining from @hahnbeelee's PR #430
- [x] Remove now unneeded v-model from all modals 

![image](https://user-images.githubusercontent.com/25535093/117560062-731fb300-b058-11eb-98c6-304f56cf1f19.png)



Remaining TODOs:

- See other items on the backlog Notion item.

### Test Plan 

1. Open the console, and use every modal listed as a TeleportModal on [this writeup](https://www.notion.so/courseplan/Modal-Refactor-fbbea3dfda52468db4719f53df7d9464). Confirm using each results in no console warnings (including TourWindow) and the styling looks as expected.
2. Use the requirements bar on mobile, and confirm reset and self check course modals work as expected with their z-indices.
